src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5168,4 +5168,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_FieldlikeEventsInRoStruct" xml:space="preserve">
     <value>Field-like events are not allowed in readonly structs.</value>
   </data>
+  <data name="IDS_FeatureRefExtensionMethods" xml:space="preserve">
+    <value>reference extension methods</value>
```

Comment:
>reference extension methods [](start = 11, length = 27)

Is this how we call this feature? This could be quite confusing, ```reference``` word can mean several different things.